_posts/2016-05-09-introducting-phpstorm-integration.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Introducing PHPStorm integration"
+date: 2016-05-09 10:00:00
+author: "@agallou"
+categories: release
+---
+
+Introduction
+------------
+
+Until recently, atoum were not integrated in Jetbrain's IDE, [PhpStorm](https://www.jetbrains.com/phpstorm/).
+
+Since several weeks, this is no longer the case : atoum's plugin for PHPStorm is live.
+
+We're gonna take a tour of the functionalities of this plugin.
+
+
+Go from test to the tested class
+--------------------------------
+
+When writing unit tests, en moreover when doiing TDD, we often switch from the test to the tested class. 
+
+The plugin adds an icon in from of the name of the tested class to go to test class. It also does the opposite and adds in icon in from of the test class.
+
+A shortcut is also available for a faster usage of this functionality.
+
+![Switch](/images/posts/2016-05-09-introducting-phpstorm-integration/switch-icon.png)
+
+
+Test identification
+-------------------
+
+The file icon is changed to add atoum's logo when the file contains a unit test.
```

Comment:
The file's icon will change when it contains a unit test: it will have the atoum's logo.